core/templates/dev/head/pages/exploration_player/TutorCardDirective.js

```diff
@@ -41,7 +41,9 @@ oppia.directive('tutorCard', [
       restrict: 'E',
       scope: {
         onSubmitAnswer: '&',
+        returnToExploration: '&',
```

Comment:
Why add this? I think the switch should be handled in "submitAnswer".

Reply:
Actually, this was not needed. I had shifted this functionality to ProgresNavDirective, will remove this.

Reply:
Done.